components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIAdmin.java

```diff
@@ -71,6 +71,16 @@ public interface APIAdmin  {
      */
     void deleteEnvironment(String tenantDomain, String uuid) throws APIManagementException;
 
+    /**
+     * Checks whether the given environment has active gateway policy deployments
```

Comment:
as per wso2 styleguide, end the java comments with a proper fullstop.

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/12201/commits/1136a795360198388c6b4bd97029c20b0f898832

Thanks and Regards